hw1/02-palindrome.js

```diff
@@ -1,3 +1,45 @@
 const elem = document.querySelector('input');
+const errorElem = document.querySelector('.inputError');
+const resultElem = document.querySelector('.displayResult');
 
 elem.addEventListener('input', handleInput);
+
+function handleInput() {
+  const inputValue = elem.value;
+  const numValue = Number(inputValue);
+
+  // Clear the error and border when input is empty
+  if (!inputValue) {
+    resultElem.textContent = '';
+    errorElem.textContent = '';
+    elem.style.borderColor = '';
+    return;
+  }
+
+  // Check if the input is a negative number
+  if (numValue < 0) {
+    errorElem.style.color = 'red';
+    elem.style.borderColor = 'red';
```

Comment:
I  like that you used a border for the error msg! I noticed that you've separated the errorElem and resultElem , which gives you control to apply different styles to each.